SiteAssets/pages/3.0/scripts/CDM.DATA.CONTROLLER.js

```diff
@@ -729,10 +729,30 @@ tposdata = {
 
         hListItem.update();
         ctx().executeQueryAsync();
-        if (callback) {
+        if (callback && callback === Function) {
             var fn = window[callback];
             fn(i);
         }
+        function onSuccess(){
+            if(callback === "addHazards"){
+                console.log("Success")
+                //let message appear after the last hazard is added
+                if(check[0] === (check[1] - 1)){
+                toastr.success(check[1] + " hazard(s) successfully added")
```

Comment:
Remove the stuff about the check - it's better to have a toast for each successful hazard creation